pkg/cloud/aws/aws.go

```diff
@@ -36,12 +36,25 @@ type AwsCloud struct {
 
 // NewAwsCloud creates an AWS cloud
 func NewAwsCloud(awsProfile, region string) (*AwsCloud, error) {
-	ctx := context.Background()
-	// Load session from shared config
-	cfg, err := config.LoadDefaultConfig(ctx,
-		config.WithRegion(region),
-		config.WithSharedConfigProfile(awsProfile),
+	var (
+		cfg aws.Config
+		err error
 	)
+	ctx := context.Background()
+	if os.Getenv("AWS_ACCESS_KEY_ID") != "" {
```

Comment:
do we need this code? 
https://aws.github.io/aws-sdk-go-v2/docs/configuring-sdk/#environment-variables - `By default, the SDK detects AWS credentials set in your environment and uses them to sign requests to AWS.`

Reply:
yes we need this. the aws sdk doc seems to be incorrect. at least for linux. please give it a test.

Reply:
if you use `config.WithSharedConfigProfile` it will ignore env vars on linux at least

Reply:
I can't reproduce it @felipemadero 
```
❯ export AWS_ACCESS_KEY_ID="XXXXXXXX"
❯ export AWS_SECRET_ACCESS_KEY="YYYYYYYYY"
❯ env | grep AWS
AWS_PAGER=
AWS_ACCESS_KEY_ID=XXXXX
AWS_SECRET_ACCESS_KEYYUYYYYYY
❯ ./bin/avalanche node create test3  --use-static-ip=false --latest-avalanchego-version=true --aws  --region us-east-1 --num-nodes=1 --devnet
✔ c5.2xlarge (default)
Creating new EC2 instance(s) on AWS...
Using existing key pair artur-us-east-1-avalanche-cli in AWS[us-east-1]
Using existing security group artur-us-east-1-avalanche-cli-us-east-1-sg in AWS[us-east-1]
Waiting for EC2 instances in AWS[us-east-1] to be provisioned...
New EC2 instance(s) successfully created in AWS!
```

Reply:
I can, it is a shame (ubuntu 22.10)

```
export AWS_ACCESS_KEY_ID="..."
export AWS_SECRET_ACCESS_KEY="..."
export AWS_SESSION_TOKEN="..."
 env | grep AWS
 AWS_SECRET_ACCESS_KEY=...
 AWS_ACCESS_KEY_ID=...
 AWS_SESSION_TOKEN=...
 ./bin/avalanche node create test3  --use-static-ip=false --latest-avalanchego-version=true --aws  --region us-east-1 --num-nodes=1 --devnet
 ✔ c5.2xlarge (default)
AWS credentials expired
Please update your environment variables AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY
Following https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html#envvars-set
Or fill in ~/.aws/credentials with updated contents following the format below
===========BEGINNING OF FILE===========
[default]
aws_access_key_id=<AWS_ACCESS_KEY>
aws_secret_access_key=<AWS_SECRET_ACCESS_KEY>
===========END OF FILE===========
More info can be found at https://docs.aws.amazon.com/sdkref/latest/guide/file-format.html#file-format-creds

Error: operation error EC2: DescribeImages, https response error StatusCode: 400, RequestID: 0ade8d20-c4b9-4517-abb8-b721d90d1e1a, api error RequestExpired: Request has expired.

```

Reply:
you are correct. It's my macosx having default AWS profile setup